tools/rna_tools/rnaz/rnaz.xml

```diff
@@ -40,6 +40,12 @@
     <outputs>
         <data name="outfile" format="fasta" />
     </outputs>
+    <tests>
+        <test>
+            <param name="input1" value="test-data/tRNA.aln"/>
```

Comment:
You don't need to specity test-data, only the name is important ... also you can add parameters here if you like and if you want to test them (you need to change the test data accordingly)